_pages/terminology-server.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Terminology Server
```

Comment:
## Added: Terminology Server page
I was also thinking about calling this "FHIR Server"